python/pylibraft/pylibraft/test/test_distance.py

```diff
@@ -79,6 +79,8 @@ def test_distance(n_rows, n_cols, inplace, metric, order, dtype):
 
     actual = output_device.copy_to_host()
 
-    actual[actual <= 1e-5] = 0.0
+    # actual[actual <= 1e-5] = 0.0
+    # if metric == "euclidean":
+    #     np.fill_diagonal(actual, 0.0)
```

Comment:
```suggestion
```

Reply:
Ah yes. Those comments can be removed. If this run passes, let me get this merged just for the sake of getting CI (and the numerical issue) fixed and then I'll follow-up with a PR to remove the commented out code.